### DISCUSSION TITLE
FUND CALL NOT BRINGING REVERT ERROR

### DISCUSSION THREAD

**Discussion by Tarreef** [Cyfrin/foundry-full-course-cu#2755] [Q&A]
![Screenshot_20241027_122432](https://github.com/user-attachments/assets/ccb14730-5cfd-40e4-b863-6d0563151c55)

after funding i am unable to see the "get estimation failed" error in remix..
rather on the terminal tab it keeps reverting an error saying "transact to FundMe.fund pending ..."
Please is anyone possibly going through the same issue.

**Comment by 10kumaranurag01** [comment]
The same happened to me today. Maybe something is up with the test net. I experienced something similar today; there might be an issue with the test net.

**Reply by Tarreef** [reply]
i have an assumption that it's related to the version of remix we're using. What version are you using

**Reply by 10kumaranurag01** [reply]
I am using the latest one, but i switched to foundry now.

**Reply by EngrPips** [reply]
It might actually be an internet connection problem, or maybe Remix is acting weird. I don't think it is a test-net issue.

**Comment by aslamdoctor** [comment]
I am facing same issue right now

**Reply by EngrPips** [reply]
That could be an internet connection problem
